helm_deploy/values-preprod-cp.yaml

```diff
@@ -20,6 +20,79 @@ generic-service:
     tlsSecretName: prison-api-cert
     modsecurity_enabled: true
 
+  allowlist:
```

Comment:
Would it be better to have more groups set up (like the internal one) if multiple services are using these clients